test/unit/credit_card_test.rb

```diff
@@ -193,6 +193,7 @@ def test_should_display_number
   def test_should_correctly_identify_card_brand
     assert_equal 'visa',             CreditCard.brand?('4242424242424242')
     assert_equal 'american_express', CreditCard.brand?('341111111111111')
+    (222100..272099).each {|bin| assert_equal 'master', CreditCard.brand?(bin.to_s + '0089600203'), "Failed with BIN #{bin}"}
```

Comment:
The majority of these are going to fail Luhn already so it can't be important. I didn't put any thought into which number to use so just pasted in the one from the certification test. I'll change it to all-ones.